agenta-web/src/components/Evaluations/EvaluationCardView/index.tsx

```diff
@@ -171,7 +175,12 @@ const EvaluationCardView: React.FC<Props> = ({
         return {scenario: evaluationScenarios[scenarioIndex], scenarioIndex}
     }, [scenarioId, evaluationScenarios])
 
-    // console.log(evaluationScenarios, isLoading)
+    useEffect(() => {
+        setEvaluationsState((prevEvaluationsState) => ({
+            ...prevEvaluationsState,
+            [evaluation.id]: {lastVisitedScenario: scenarioId},
```

Comment:
please change this line to only change the `lastVisitedScenario`, so if in the future there are other fields in this object, they are not accidentally overwritten:

```suggestion
            [evaluation.id]: {...(prevEvaluationsState[evaluation.id] || {}), lastVisitedScenario: scenarioId},
```